packages/blockprotocol/core.d.ts

```diff
@@ -9,6 +9,7 @@ type DistributedOmit<T, K extends PropertyKey> = T extends T
 export type BlockVariant = {
   description?: string | null;
   displayName?: string | null;
+  examples?: JSONObject[] | null;
```

Comment:
Update the variants type with examples array.

Reply:
I guess it’s fine to change the spec here, especially in a non-breaking way. We are still pre-0.1.0, so we don’t have to be too strict. We can release a new package version after making a few small batch updates like that.